.github/workflows/main.yml

```diff
@@ -99,17 +99,53 @@ jobs:
       docker_password: "${{ secrets.DOCKER_PASSWORD }}"
 
   trivy:
-    needs:
-      - app-version
-      - build-and-push-image
-    uses: SierraSystems/reusable-workflows/.github/workflows/trivy-scan-openshift-image.yml@main
-    with:
-      imagestream_name: ${{ github.event.inputs.app_name }}
-      image_tag: "${{ needs.app-version.outputs.app-version }}"
-    secrets:
-      openshift_external_repository: "${{ secrets.OPENSHIFT_EXTERNAL_REPOSITORY_SILVER }}"
-      openshift_namespace: "${{ secrets.OPENSHIFT_LICENSE_PLATE_SILVER }}-tools"
-      openshift_sa_name: "${{ secrets.OPENSHIFT_SA_PIPELINE_PASSWORD_SILVER }}"
-      openshift_sa_password: "${{ secrets.OPENSHIFT_SA_PIPELINE_TOKEN_SILVER }}"
-      openshift_server_url: "${{ secrets.OPENSHIFT_SERVER_URL_SILVER }}"
-      openshift_token: "${{ secrets.OPENSHIFT_SA_PIPELINE_TOKEN_SILVER }}"
+    name: Build & Scan Image
+    runs-on: ubuntu-20.04
+    env:
+    steps:
+      - name: Checkout code
```

Comment:
line #3 - #132 are copied from dev-dps-email-poller.yml to pull image from OpenShift.